app/Services/WMURetriever.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Services;
+
+use App\Plot;
+use Illuminate\Support\Facades\Http;
+
+class WMURetriever
+{
+    /**
+     * @var \App\Plot
+     */
+    protected $plot;
+
+    /**
+     * Create the retriever.
+     *
+     * @return void
+     */
+    public function __construct(Plot $plot)
+    {
+        $this->plot = $plot;
+    }
+
+    /**
+     * Get the WMU for the plot.
+     *
+     * @return void
+     */
+    public function setWMU()
+    {
+        return rescue(function () {
+            $response = Http::get('localhost:5000', [
```

Comment:
You probably want to specify the scheme since the request is not internal (not within the same application). Laravel adds the scheme automatically but this is more expressive (if we ever add ssl we know what to change in the future).
```suggestion
            $response = Http::get('http://localhost:5000', [
```